src/main/java/io/github/jbellis/brokk/gui/dialogs/SettingsDialog.java

```diff
@@ -238,7 +238,7 @@ public static boolean showStandaloneDataRetentionDialog(IProject project, Frame
 
         var dialog = new JDialog(owner, "Data Retention Policy Required", true);
         dialog.setDefaultCloseOperation(JDialog.DO_NOTHING_ON_CLOSE);
-        dialog.setSize(600, 300); // Adjusted size
+        dialog.setSize(600, 350); // Adjusted size
```

Comment:
On my machine, not all the content was shown with 300 height, so increased a bit:
![image](https://github.com/user-attachments/assets/83cfc0df-7973-402b-b7ff-97f5541aad57)

Trying to solve with `dialog.pack()` made the dialog too wide because it seems pack() tends to keep labels (with html content in this case) on one line only.